library/kotlin/src/io/envoyproxy/envoymobile/filters/FilterDataStatus.kt

```diff
@@ -5,14 +5,16 @@ import java.nio.ByteBuffer
 /*
  * Status to be returned by filters when transmitting or receiving data.
  */
-sealed class FilterDataStatus<T : Headers> {
+sealed class FilterDataStatus<T : Headers>(
+  val status: Int
```

Comment:
Introduce a "static" constant value here that maps to Envoy's enum. It's not possible to directly set these from the native values, but I'd like to add a runtime check and/or test that verifies they're the same.